web/src/main/java/org/cbioportal/web/util/ScientificSmallDataBinner.java

```diff
@@ -29,6 +30,8 @@ public ScientificSmallDataBinner(DataBinHelper dataBinHelper) {
             .filter(d -> d.compareTo(new BigDecimal("0")) != 0)
             .collect(Collectors.toList());
 
+		Collections.sort(exponents);
```

Comment:
Indentation looks a bit off here.

Reply:
Checked, It looks right to me. try comparing to below lines( not the lines above as they are methods)

Reply:
This is what I see on github:

<img width="447" alt="indentation" src="https://user-images.githubusercontent.com/3604198/89576499-5fab9700-d7fd-11ea-901e-c8d13a0d3406.png">

Ideally it should be aligned with the line starting with `List...` above but it looks off. Seems like tab vs space issue to me. Anyway, not a big deal.

Reply:
Got it, updated. Thanks!